frontend-sdk/README.md

```diff
@@ -0,0 +1,151 @@
+# hanko-frontend-sdk
+
+This package utilizes the [Hanko API](https://github.com/teamhanko/hanko/blob/main/backend/README.md) to provide
+functionality that allows an easier UI integration. It is meant for use in browsers only.
+
+## Installation
+
+```shell
+# npm
+npm install @teamhanko/hanko-frontend-sdk
+
+# yarn
+yarn add @teamhanko/hanko-frontend-sdk
+
+# pnpm
+pnpm install @teamhanko/hanko-frontend-sdk
+```
+
+## Usage
+
+Import as a module:
+
+```typescript
+import { Hanko } from "@teamhanko/hanko-frontend-sdk"
+
+const hanko = new Hanko("http://localhost:3000")
+```
+
+With a script tag via CDN:
+
+```html
+<script src="https://unpkg.com/@teamhanko/hanko-frontend-sdk/dist/sdk.umd.js"></script>
```

Comment:
We changed our default CDN to JSDeliver for performance reasons if I'm not mistaken

Reply:
Thanks, changed to JSDelivr here and in hanko-js.